core/src/main/kotlin/com/exactpro/th2/read/db/impl/grpc/DataBaseReaderGrpcServer.kt

```diff
@@ -148,10 +151,22 @@ class DataBaseReaderGrpcServer(
     companion object {
         private val LOGGER = KotlinLogging.logger { }
 
-        internal fun DbPullResetState.toResetState(): ResetState = ResetState(
-            if(hasAfterDate()) afterDate.toInstant() else null,
-            //TODO: maybe use another protobuf type or check date unit
-            if(hasAfterTime()) LocalTime.ofInstant(afterTime.toInstant(), ZoneOffset.UTC) else null
-        )
+        internal fun DbPullResetState.toResetState(): ResetState {
```

Comment:
I think that we should move this method to all others that convert gRPC to an internal model. Also, for consistency purposes, I would name it `toModel`. But I do not insist on that and if you want to keep the current code state I have no objections